heareval/tasks/pipeline.py

```diff
@@ -186,25 +187,6 @@ def slugify_file_name(relative_path: str):
         slug_text = slug_text.replace("-", "_negative_")
         return f"{slugify(slug_text)}"
 
```

Comment:
I accidentally remove split_key in this PR. I add it back in https://github.com/neuralaudio/hear-preprocess/pull/8